assignments/array-methods.js

```diff
@@ -54,30 +54,46 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 {"id":50,"first_name":"Shell","last_name":"Baine","email":"sbaine1d@intel.com","shirt_size":"M","company_name":"Gabtype","donation":171}];
 
 // ==== Challenge 1: Use .forEach() ====
-// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
+// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName.
 let fullName = [];
+runners.foreach(` ${ runners[i]["first_name"] } ${ runners[i]["last_name"] }`);
+
+runners.foreach(fullName.push(runners[i].first_name + " " + runners[i].last_name ));
+
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
-console.log(allCaps); 
+allCaps = runners.map(function(items) {return items.first_name.toUpperCase();});
+console.log(allCaps);
 
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
 let largeShirts = [];
+largeShirts = runners.filter((list) => {return list.shirt_size === "L" ;});
```

Comment:
`runners.filter((list) => {return list.shirt_size === "L" ;});` could also be written as `runners.filter(list => list.shirt_size === "L" );` (you don't need the return statement if it's written on the same line because it's implied. Parentheses are also optional on an arrow function if you're only passing 1 param.